lib/command/base.rb

```diff
@@ -45,12 +45,12 @@ def initialize(config)
       @config = config
     end
 
-    def self.all_commands # rubocop:disable Metrics/MethodLength
+    def self.all_commands # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity
       Dir["#{__dir__}/**/*.rb"].each_with_object({}) do |file, result|
         content = File.read(file)
 
         classname = content.match(/^\s+class (\w+) < (?:.*Base)(?:$| .*$)/)&.captures&.first
-        next unless classname
+        next if classname.nil? || classname == "Base"
```

Comment:
Commands from `Base` classes should not be generated

Reply:
We were already skipping those in the regex at the line above: `/^\s+class (\w+) < (?:.*Base)(?:$| .*$)/`

Maybe we should either expand it to include the new case, or if that's too complex, remove it and handle the old cases here as well?

Reply:
Changed regex to include the new case